sonic_y_cable/y_cable.py

```diff
@@ -1235,6 +1235,14 @@ def get_firmware_version(physical_port, target):
         result["empty_slot1"] = True if slot_status & 0x04 else False
         result["empty_slot2"] = True if slot_status & 0x40 else False
 
+        version_build_slot1 = version_slot1 + build_slot1
+        version_build_slot2 = version_slot2 + build_slot2
+
+        result["version_active"] = version_build_slot1 if slot_status & 0x01 else version_build_slot2
+        result["version_inactive"] = version_build_slot2 if slot_status & 0x01 else version_build_slot1
+        result["version_next"] = version_build_slot1 if slot_status & 0x02 else version_build_slot2
```

Comment:
These new fields are intended to supplant the fields on lines 1227-1236, simplifying the resulting structure. Is there a reason you did not delete lines 1227-1236?

Reply:
just thinking to keep them for now, until xcvrd has the change where it does not look for old fields and fetches these new fields, otherwise we might see regression, if we remove the old fields without making the change in xcvrd. Once xcvrd is updated I will have another PR to remove these old fields. 
The other approach is to make the change in xcvrd and y_cable.py and make the submodule ptr update together in one PR for both sonic-platform-common and sonic-platform-daemons, so that nothing breaks. 
What do you suggest ?

Reply:
Makes sense to allow for a smooth transition. Maybe add a TODO comment to remove them eventually?